### PR TITLE
[WIP] Prepare first release

### PR DESCRIPTION
**Pull request by dpordomingo**
Solves first subtask of https://github.com/bblfsh/dashboard/issues/46 and partially its 2nd and 3rd

## Pending:
- **tests**
- load the gist by url

### CODE REVIEW

**Comment by dpordomingo** [comment]
CI fails because I changed many things without updating the tests... I know... I'll fix it asap

File: server/server.go [hunk 1]
```diff
@@ -88,6 +89,25 @@ func (s *Server) clientForRequest(req parseRequest) (protocol.ProtocolServiceCli
 	return newClient(req.ServerURL)
 }
 
+func (s *Server) LoadGist(ctx *gin.Context) {
```
Comment (smacker) [inline server/server.go]: why do you need proxy?
Github allows to do cross origin requests to gist.githubusercontent.com

Comment (dpordomingo) [reply server/server.go]: That's a quite good approach. Since you're resuming this issue, feel free to proceed that way

**Comment by dpordomingo** [comment]
superseded by https://github.com/bblfsh/dashboard/pull/73

**Pull request closed without merging**